ibm_catalog.json

```diff
@@ -0,0 +1,234 @@
+                "description": "Yes" 
+              },
+              {
+                "title": "Virtual Server Instances for every subnet",
+                "description": "Yes" 
+              },
+              {
+                "title": "Increases security with Key Management",
+                "description": "Yes" 
+              },
+              {
+                "title": "Collects and stores Internet Protocol (IP) traffic information with Activity Tracker and Flow Logs",
+                "description": "Yes" 
+              },
+              {
+                "title": "Securely connects to multiple networks with a client-to-site/site-to-site virtual private network",
+                "description": "Yes"
+              },
+              {
+                "title": "Simplifies risk management and demonstrates regulatory compliance with Financial Services",
+                "description": "Yes" 
+              },
+              {
+                "title": "Uses an edge VPC for secure access through the public internet",
+                "description": "Yes" 
+              },
+              {
+                "title": "Uses floating IP addresses for access through the public internet",
+                "description": "No" 
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "caption": "Wazi as a service VSI on VPC Landing zone - Standard variation",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-zvsi/standard_quickstart/reference-architecture/Standard-variation.svg",
+                  "type": "image/svg+xml"
+            },
+            "description": "Standard Variation"
+           }
+           ]
+          },
+          "configuration": [
+            {
+              "custom_config": {
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "type": "json_editor"
+              },
+              "key": "override_json_string"
+            },
+            {
+              "custom_config": {
+                "config_constraints": {
+                  "generationType": "2"
+                },
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "type": "vpc_region"
+              },
+              "key": "region",
+              "required": true,
+              "type": "string"
+            }
+          ],
+          "install_type": "fullstack"
+        },
+        {
+          "label": "Quickstart",
+          "name": "quickstart",
+          "working_directory": "solutions/quickstart",
+          "compliance": {
+            
+          },
+          "iam_permissions": [
+            {
+              "service_name": "is.vpc",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Operator"
+              ]
+            }
+          ],
+          "release_notes_url": {
```

Comment:
The `release_notes_url` value should be a string

Reply:
Done.